### PR TITLE
SNOW-1907 : Fetch Client details through snowflake-data-loader script for snapshot and replication data

### PR DESCRIPTION
**Pull request by utkarsh-unravel**
[SNOW-1907 : Fetch Client details through snowflake-data-loader script for snapshot and replication data](https://unraveldata.atlassian.net/browse/SNOW-1907)

### CODE REVIEW

File: script/snow_share/health_check_procedure.sql [hunk 1]
```diff
@@ -567,6 +572,7 @@ GRANT SELECT ON TABLE DATABASE_REPLICATION_USAGE_HISTORY to share S_SECURE_SHARE
 GRANT SELECT ON TABLE REPLICATION_GROUP_USAGE_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE SNOWPIPE_STREAMING_FILE_MIGRATION_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE QUERY_HISTORY to share S_SECURE_SHARE;
+GRANT SELECT ON TABLE SESSIONS to share S_SECURE_SHARE_4825_2;
```
Comment (viswa-unravel) [inline script/snow_share/health_check_procedure.sql:575]: This is wrong, why it is 4825_2, then share will not work

**Review by viswa-unravel — COMMENTED** [review]
Pls fix this issues

File: script/snow_share/health_check_procedure.sql (hunk 1 above)
Comment (utkarsh-unravel) [reply script/snow_share/health_check_procedure.sql]: @viswa-unravel that was a typo. corrected it now.

**Review by viswa-unravel — APPROVED** [review]
LGTM

**Pull request merged**